sql/core/src/test/resources/sql-tests/inputs/pgSQL/with.sql

```diff
@@ -1190,7 +1183,9 @@ SELECT * FROM parent;
 --DROP RULE y_rule ON y;
 
 -- check that parser lookahead for WITH doesn't cause any odd behavior
+-- [TODO] WITH should be a reserved keyword?
```

Comment:
Hi, @maropu . 
Could you give us some advice on this?

Reply:
yea, already reaserved in spark (ansi=true). https://github.com/apache/spark/blob/master/docs/sql-keywords.md

Reply:
Probably, we can always set ansi=true for these PostgreSQL ported tests by set ansi=true in SQLQueryTestSuite.

Reply:
+1 for that.